code/build.R

```diff
@@ -456,7 +456,7 @@ m =
            lwd = 3,
            # scale = 5,
            alpha = 1,
-           popup.vars = popup.vars
```

Comment:
True that, using segment level data means the variables available changed. I should have spotted that in your previous PR. Apologies and many thanks for the fast fix!